avaframe/ana4Stats/getStats.py

```diff
@@ -73,11 +73,26 @@ def extractMaxValues(inputDir, cfgMain, avaDir, nameScenario=''):
     peakFiles = fU.makeSimDict(inputDir, varPar, avaDir)
     nSims = len(peakFiles['simName'])
 
-    # initialize dictionary to save values
+    # load parameter variation values to check if they include default value
+    varParValues = cfgMain['PARAMETERVAR']['varParValues']
+    if ':' in varParValues:
+        itemsInput = varParValues.split(':')
+        itemsRaw = np.linspace(float(itemsInput[0]), float(itemsInput[1]), int(itemsInput[2]))
+    else:
+        itemsRaw = varParValues.split('_')
+        itemsRaw = np.array(itemsRaw,dtype=float)
+    flagValue = False
+    if float(cfgMain['DEFVALUES'][cfgMain['PARAMETERVAR']['varPar']]) in itemsRaw:
```

Comment:
This is hard to read: since the `cfgMain['PARAMETERVAR']['varPar']` is used in multiple places, maybe break it out like the varParValues above?

Reply:
done